Makefile

```diff
@@ -163,7 +167,7 @@ $(bbl): $(pk_srcdir) $(vmlinux_stripped) $(dtb)
 	cd $(pk_wrkdir) && $</configure \
 		--host=$(target) \
 		--with-payload=$(vmlinux_stripped)
-	CFLAGS="-mabi=$(ABI) -march=$(BBL_ISA)" $(MAKE) -C $(pk_wrkdir)
+	CFLAGS="-mabi=$(ABI) -march=$(BBL_ISA) $(LITEX_MODE)" $(MAKE) -C $(pk_wrkdir)
```

Comment:
@scanakci why are we changing the march flag here?

Reply:
I am not actually changing march flag. I add an extra argument as part of CFLAGS. When someone compiles with "make bbl LITEX_MODE=-DLITEX_MODE", LITEX_MODE macro will be defined and activates necessary lines in the modified firmware files. Otherwise, $LITEX_MODE will not have any impact and generate the original bbl without LITEX support.

Reply:
Oh, you're right. I read it wrong.